app/views/tracks/exercises/show/_iteration_section.html.haml

```diff
@@ -0,0 +1,120 @@
+/ TODO: Convert everything to React
```

Comment:
@kntsoriano This file will be changing to React and working off websockets. I've written it all out in a way that I hope makes it easy, and I've added system tests to help too. I'll write this up properly tomorrow.